.github/workflows/e2e-tests.yml

```diff
@@ -37,6 +37,13 @@ jobs:
       fail-fast: false
       matrix:
         network-setting: [shared-vnet, peered-vnet,  dynamic-ip-allocation]
+        include:
+          - network-setting: shared-vnet
+            enable-traffic-manager: true
```

Comment:
Why do we only enable traffic manager in shared-vnet scenario?

Reply:
we will support others in a separate PR. i don't want to put too many changes in one PR.